src/explore-education-statistics-common/src/components/AccordionSection.tsx

```diff
@@ -81,7 +82,7 @@ const AccordionSection = ({
             headingTag,
             {
               className: classes.sectionHeading,
-              id: heading.toLowerCase().split(' ').join('-'),
+              id: `section-${kebabCase(heading)}`,
```

Comment:
As discussed in person, think there's a few things we need to sort out here:

- `anchorLinkUrl` should be changed to callback so that we can provide an id parameter to the consumer (so it doesn't have to figure out what the anchor ID should be) e.g.
  
  ```tsx
  <EditableAccordionSection 
    anchorLinkUrl={id => `${PublicAppUrl}/methodology/${currentMethodology.slug}#${id}`}
  >
  ```
- We should check with Cam if we human-readable IDs are a necessity. We might want to use guids instead as these are more stable (analysts can change section titles, thus breaking links) and they are unique (analysts may create sections with the same title and this would cause ambiguity).
- We should preferably use the `headingId` that is already set in `AccordionSection` instead of making a new id. This will be dependent on the previous point about human-readable ids.
- We'll need to update the public frontend accordingly (see `MethodologyPage`). Currently, it's using the section's `order` for the id of each, but we should really use the ID.